python/tests/test_topology.py

```diff
@@ -5858,6 +5864,142 @@ def test_many_multiroot_trees_recurrent_mutations(self):
             self.verify_branch_mutations(ts, mutations_per_branch)
 
 
+class TestMutationTime(unittest.TestCase):
+    """
+    Tests that mutation time is correctly specified, and that we correctly
+    recompute it with compute_mutation_times.
+    """
+
+    seed = 42
+
+    def verify_times(self, ts):
+        time = tsutil.compute_mutation_times(ts)
+        tables = ts.tables
+        print(time - tables.mutations.time)
```

Comment:
Stray print here

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/672/commits/93e65c6ddc6f4768c1ac07576e62979bb5a04604